README.md

```diff
@@ -1,14 +1,14 @@
-# HarbourBridge: Turnkey Postgres-to-Spanner Evaluation
+# HarbourBridge: Turnkey Postgres-to-Spanner / MySQL-to-Spanner Evaluation
```

Comment:
the space after and before / looks weird, remove the white space, this applies to everywhere there is /.
example, Postgres-to-Spanner / MySQL-to-Spanner   ---> Postgres-to-Spanner/MySQL-to-Spanner